kivy/core/window/__init__.py

```diff
@@ -732,6 +732,19 @@ def _get_focus(self):
     def _set_cursor_state(self, value):
         pass
 
+    def set_system_cursor(self, cursor_name):
+        '''Set cursor to a SDL_SystemCursor
```

Comment:
Run the docstring e.g. through http://rst.ninjs.org and fix the formatting, please :)